src/components/common/account/info/InfoButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import theme from '../../../../theme'
+import { Button, Link } from '@mui/material'
+
+interface Props {
+    buttonUrl: string
+    buttonText: string
+}
+
+export const InfoButton = (props: Props) => {
+    const { buttonUrl, buttonText } = props
+    return (
+        <>
+            <Button
+                sx={{
+                    textTransform: 'none',
+                    height: '39px',
+                    textAlign: 'center',
+                    backgroundColor: theme.palette.primary.main,
+                    borderRadius: 0,
+                    margin: '5px 25px',
+                }}
+            >
+                <Link
+                    href={buttonUrl}
+                    underline="none"
+                    sx={{
+                        color: theme.palette.text.primary,
+                        fontWeight: 'lighter',
+                        fontSize: '16px',
+                        lineHeight: '27px',
+                        width: '100%',
+                    }}
+                >
+                    {buttonText}
+                </Link>
+            </Button>
+        </>
+    )
+}
```

Comment:
hejka, tu mała uwaga :) w moim feature tez zrobiłam Button uniwersalny więc jak zmergujemy bedzie mozna to połączyc aby uzywac Buttona na resorach :)